### PR TITLE
Fix: updating basic dsl for fork and duplicate feature cypress specs

### PR DESCRIPTION
**Pull request by marks0351**
---
name: "Pull Request"
about: Create a pull request to submit new contributions to appsmith
title: "[FIX/FEATURE]"
labels:
assignees:
---

> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

## Description

> Please include a summary of the changes and which issue has been fixed. Please also include relevant motivation
> and context. List any dependencies that are required for this change.

Fixes # (issue)

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: fix/fork-duplicate-spec 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 51.71 **(0.02)** | 32.95 **(0.02)** | 29.67 **(0.03)** | 52.2 **(0.03)**
 :green_circle: | app/client/src/utils/WidgetPropsUtils.tsx | 65.43 **(1.23)** | 53.28 **(0.97)** | 56.94 **(2.77)** | 64.87 **(1.29)**</details>

### CODE REVIEW

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

**Comment by hetunandu** [comment]
@marks0351 Can you explain this change?

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Applications/ForkApplication_spec.js [hunk 1]
```diff
@@ -14,9 +14,11 @@ describe("Fork application across orgs", function() {
   it("Check if the forked application has the same dsl as the original", function() {
     const appname = localStorage.getItem("AppName");
     cy.SearchEntityandOpen("Input1");
+    cy.intercept("PUT", "/api/v1/layouts/*/pages/*").as("inputUpdate");
     cy.get(widgetsPage.defaultInput).type("A");
     cy.get(commonlocators.editPropCrossButton).click({ force: true });
-    cy.wait("@updateLayout").then((response) => {
+    cy.wait(2000);
```
Comment (mohanarpit) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Applications/ForkApplication_spec.js:20]: Why wait for 2 secs here when we are already waiting for the API to complete in the next line?

Comment (marks0351) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Applications/ForkApplication_spec.js]: thats right. was trying out something and forgot to remove it. removing it now.

**Review by NandanAnantharamu — APPROVED** [review]
Approved for Test run

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

**Pull request merged**